.github/workflows/unit_test.yml

```diff
@@ -55,12 +53,11 @@ jobs:
           poe lint
           poe lint-warnings
       - name: Test
-        if: contains(github.event.comment.body, '/run_tests')  # check if the comment calls the tests execution
```

Comment:
But now you should replace this with the label query (and schedule, whenever there will be a schedule event).